public/component_types/interfaces.ts

```diff
@@ -35,6 +44,8 @@ export interface IComponentInput {
 export interface IComponentField {
   label: string;
   type: FieldType;
+  name: string;
+  value?: FieldValue;
```

Comment:
Just wondering if we want to have initial values in some cases, lets say we want to make a custom cohere components of sort, will this be the field that gets autopopulated by the formik initial values?

Reply:
Yes, right now I populate with initial values based on the field type - see `getInitialValue()`. Rudimentary for now but will expand as add more types or have some more intuitive defaults.

Reply:
Perhaps some component types may have some defined initial values too. Not adding that for now until that's a necessity.